notebooks/community/model_garden/model_garden_keras_yolov8.ipynb

```diff
@@ -0,0 +1,754 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.    model.wait()
</pre><p>This is not needed</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2159/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
Thanks!